policyengine_us/parameters/gov/states/il/idoa/bap/income_limit.yaml

```
@@ -0,0 +1,26 @@
+description: Illinois limits its Benefits Access Program to households with gross income below this amount, based on household size.
```

also check capitalization on this throughout
```suggestion
description: Illinois limits its Benefit Access Program to households with gross income below this amount, based on household size.
```